with-graphql/graphql/api_spacex_land.graphql

```diff
@@ -0,0 +1,1360 @@
+  description: String
+  diameter: spacex_Distance
+  dry_mass_kg: Int
+  dry_mass_lb: Int
+  first_flight: String
+  heat_shield: spacex_DragonHeatShield
+  height_w_trunk: spacex_Distance
+  id: ID
+  launch_payload_mass: spacex_Mass
+  launch_payload_vol: spacex_Volume
+  name: String
+  orbit_duration_yr: Int
+  pressurized_capsule: spacex_DragonPressurizedCapsule
+  return_payload_mass: spacex_Mass
+  return_payload_vol: spacex_Volume
+  sidewall_angle_deg: Float
+  thrusters: [spacex_DragonThrust]
+  trunk: spacex_DragonTrunk
+  type: String
+  wikipedia: String
+}
+
+type spacex_Distance {
+  feet: Float
+  meters: Float
+}
+
+type spacex_DragonHeatShield {
+  dev_partner: String
+  material: String
+  size_meters: Float
+  temp_degrees: Int
+}
+
+type spacex_Mass {
+  kg: Int
+  lb: Int
+}
+
+type spacex_Volume {
+  cubic_feet: Int
+  cubic_meters: Int
+}
+
+type spacex_DragonPressurizedCapsule {
+  payload_volume: spacex_Volume
+}
+
+type spacex_DragonThrust {
+  amount: Int
+  fuel_1: String
+  fuel_2: String
+  pods: Int
+  thrust: spacex_Force
+  type: String
+}
+
+type spacex_Force {
+  kN: Float
+  lbf: Float
+}
+
+type spacex_DragonTrunk {
+  cargo: spacex_DragonTrunkCargo
+  trunk_volume: spacex_Volume
+}
+
+type spacex_DragonTrunkCargo {
+  solar_array: Int
+  unpressurized_cargo: Boolean
+}
+
+type spacex_Info {
+  ceo: String
+  coo: String
+  cto_propulsion: String
+  cto: String
+  employees: Int
+  founded: Int
+  founder: String
+  headquarters: spacex_Address
+  launch_sites: Int
+  links: spacex_InfoLinks
+  name: String
+  summary: String
+  test_sites: Int
+  valuation: Float
+  vehicles: Int
+}
+
+type spacex_Address {
+  address: String
+  city: String
+  state: String
+}
+
+type spacex_InfoLinks {
+  elon_twitter: String
+  flickr: String
+  twitter: String
+  website: String
+}
+
+input spacex_CoresFind {
+  asds_attempts: Int
+  asds_landings: Int
+  block: Int
+  id: String
+  missions: String
+  original_launch: Date
+  reuse_count: Int
+  rtls_attempts: Int
+  rtls_landings: Int
+  status: String
+  water_landing: Boolean
+}
+
+type spacex_Core {
+  asds_attempts: Int
+  asds_landings: Int
+  block: Int
+  id: ID
+  missions: [spacex_CapsuleMission]
+  original_launch: Date
+  reuse_count: Int
+  rtls_attempts: Int
+  rtls_landings: Int
+  status: String
+  water_landing: Boolean
+}
+
+input spacex_HistoryFind {
+  end: Date
+  flight_number: Int
+  id: ID
+  start: Date
+}
+
+type spacex_History {
+  details: String
+  event_date_unix: Date
+  event_date_utc: Date
+  id: ID
+  links: spacex_Link
+  title: String
+  flight: spacex_Launch
+}
+
+type spacex_Link {
+  article: String
+  reddit: String
+  wikipedia: String
+}
+
+type spacex_Launch {
+  details: String
+  id: ID
+  is_tentative: Boolean
+  launch_date_local: Date
+  launch_date_unix: Date
+  launch_date_utc: Date
+  launch_site: spacex_LaunchSite
+  launch_success: Boolean
+  launch_year: String
+  links: spacex_LaunchLinks
+  mission_id: [String]
```

Comment:
You could create a link with `spacex_Mission` here

Reply:
I made a change to add it --- but it's returning `null`, perhaps because I had to change the id type